Lawyer Screms

WonderPsycho opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

# About The Pull Request

adds in new funny alternate lawyer screams to many of the lawyers default clothes, both suit jackets and suits! (also useful for those who want to roleplay as Pheonix Wright or Makoto Neagi for the memes on station, i made your dreams come true)

## Why It's Good For The Game

Lawyer:OBJECTION! OBJECTION! OBJECTION! OBJECTION! OBJECTION! OBJECTION! OBJECTION! NO THAT'S WRONG!!

## Changelog

:cl:WonderPsycho
add:new shitposty Lawyer screams
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

nednaZ reviewed (changes requested):
Audio quality on "NOTHATSWRONG1.ogg" and "NOTHATSWRONG2.ogg" is too low and has a lot of background music still in it.

You don't need to add the alternative scream to every outfit, just do it to the base item '/obj/item/clothing/under/rank/civilian/lawyer'

Please fix the spelling error in the folder name for readability. "Screams", not "Screms"

WonderPsycho commented:
> Audio quality on "NOTHATSWRONG1.ogg" and "NOTHATSWRONG2.ogg" is too low and has a lot of background music still in it.
> 
> You don't need to add the alternative scream to every outfit, just do it to the base item '/obj/item/clothing/under/rank/civilian/lawyer'
> 
> Please fix the spelling error in the folder name for readability. "Screams", not "Screms"

i intentionally named the folder "lawyer screms" as a joke, but i will change the name of the folder to "Lawyer Screams" for readability

WonderPsycho commented:
renamed the Lawyer Screms folder to Lawyer Screams for readability, and  changed the civilian.dm file to have it play sounds on the lawyer outfit base file instead of all variations of it, also fixed the audio of NOTHATSWRONG1.ogg and changed it to NoThatsWrong1New.ogg, also removed NOTHATSWRONG2.ogg cause it was unnecessary for to have that file since the DR video game series is lesser known then the Ace Attorney Series anyways

WonderPsycho commented:
oh jeez a single run test fail

nednaZ commented on monkestation/code/modules/clothing/suits/jobs.dm:
```diff
@@ -0,0 +1,15 @@
+								'monkestation/sound/voice/screams/misc/Lawyer screams/OBJECTION1.ogg',
+								'monkestation/sound/voice/screams/misc/Lawyer screams/Objection2.ogg')
+
+/obj/item/clothing/suit/toggle/lawyer/purple
```
You don't need this one at all, since the parent type covers it

nednaZ commented on monkestation/code/modules/clothing/suits/jobs.dm:
```diff
@@ -0,0 +1,15 @@
+								'monkestation/sound/voice/screams/misc/Lawyer screams/Objection2.ogg')
+
+
+/obj/item/clothing/suit/toggle/lawyer/black
```
Nor this one

nednaZ reviewed (changes requested):
"NoThatsWrong1New.ogg" still sounds really bad.
Just get rid of it at this point and keep the objections.
